lib/generators/good_job/templates/update/migrations/03_add_index_good_jobs_concurrency_key_created_at.rb.erb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIndexGoodJobsConcurrencyKeyCreatedAt < ActiveRecord::Migration<%= migration_version %>
+  def change
+    add_index :good_jobs, [:concurrency_key, :created_at]
```

Comment:
Always make sure this is done concurrently

Reply:
Sure thing, done in 8b69fd99fa388b7e50967ab6d735cd6a7762ab39